tests/53groups/03local.pl

```diff
@@ -100,31 +100,35 @@
    };
 
 
-push our @EXPORT, qw( matrix_invite_group_users matrix_accept_group_invite matrix_get_joined_groups matrix_remove_group_self );
+push our @EXPORT, qw( matrix_invite_group_user matrix_accept_group_invite matrix_get_joined_groups matrix_leave_group );
 
 
-=head2 matrix_invite_group_users
+=head2 matrix_invite_group_user
+
+   matrix_invite_group_user( $inviter, $group_id, $invitee )
 
 Invite user to group
 
 =cut
 
-sub matrix_invite_group_users
+sub matrix_invite_group_user
 {
    my ( $inviter, $group_id, $invitee ) = @_;
 
    my $invitee_id = $invitee->user_id;
 
    do_request_json_for( $inviter,
       method  => "PUT",
-      uri     => "/unstable/groups/$group_id/admin/users/invite/$invitee_id",
+      uri     => "/r0/groups/$group_id/admin/users/invite/$invitee_id",
       content => {},
    );
 }
 
 
 =head2 matrix_remove_group_users
 
+   matrix_remove_group_users( $inviter, $group_id, $invitee )
```

Comment:
ok you can't have `matrix_invite_group_user` but `matrix_remove_group_user`**s**.

`user` feels more intituitive; please rename this one too.